<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>373</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dynamic Loader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
      </property>
      <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>DLL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="file_browser">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="file_selector">
        <property name="maximumSize">
         <size>
          <width>214</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="ProcessSelector" name="process_selector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>247</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" rowspan="8" colspan="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QPushButton" name="inject_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Inject</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QCheckBox" name="automatic_reinjection">
        <property name="text">
         <string>Automatic Re-Injection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QCheckBox" name="dll_copy">
        <property name="text">
         <string>DLL Copy</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Manual Mapping</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3" rowspan="10">
     <widget class="QTreeWidget" name="module_list">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Architecture</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>ac_client.exe</string>
       </property>
       <property name="text">
        <string>process</string>
       </property>
       <property name="text">
        <string>x64</string>
       </property>
       <item>
        <property name="text">
         <string>assaultcube.dll</string>
        </property>
        <property name="text">
         <string>module</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProcessSelector</class>
   <extends>QComboBox</extends>
   <header>processselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
